homework/tests/Homework.spec.js

```diff
@@ -0,0 +1,145 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+  test.describe(version + ': Add', () => {
+    test('1. Add 2 and 3 results in 5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
Since this is a common step for every test case, you could move this to a `beforeEach` hook above all tests

Reply:
Done